tool/qiime2/diversity.visualizers.alpha_rarefaction.cwl

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env cwl-runner
+
+cwlVersion: v1.0
+class: CommandLineTool
+id: qiime2.plugins.diversity.visualizers.alpha_rarefaction
+requirements:
+  DockerRequirement:
+    dockerPull: qiime2/core:2019.4
+    dockerImageId: qiime2/core:2019.4
+    dockerOutputDirectory: /home/qiime2
+label: Alpha rarefaction curves
+doc: Generate interactive alpha rarefaction curves by computing rarefactions between `min_depth` and `max_depth`. The number of intermediate depths to compute is controlled by the `steps` parameter, with n `iterations` being computed at each rarefaction depth. If sample metadata is provided, samples may be grouped based on distinct values within a metadata column.
+
+baseCommand: qiime
+arguments:
+  - diversity
+  - alpha-rarefaction
+
+inputs:
+  table:
+    label: table
+    doc: Feature table to compute rarefaction curves from.
+    type: File
+    inputBinding:
+      prefix: --i-table
+  phylogeny:
+    label: phylogeny
+    doc: Optional phylogeny for phylogenetic metrics.
+    type: File
+    inputBinding:
+      prefix: --i-phylogeny
+  max_depth:
+    default: null
+    doc: The maximum rarefaction depth. Must be greater than min_depth.
+    type: long
+    inputBinding:
+      prefix: --p-max-depth
+  metrics:
```

Comment:
In the [spec of v1.1](https://www.commonwl.org/v1.1/CommandLineTool.html#CommandLineBinding),
```text
If the array is empty, it does not add anything to command line.
```
I guess it is same as v1.0.

Therefore it can be simplified as follows:
```cwl
metrics:
  default: []
  type: string[]
  ...
```